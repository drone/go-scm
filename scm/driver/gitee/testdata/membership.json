{
  "url": "https://gitee.com/api/v5/orgs/kit101-personal/memberships/kit101",
  "active": true,
  "remark": null,
  "role": "admin",
  "organization_url": "https://gitee.com/api/v5/orgs/kit101-personal",
  "organization": {
    "id": 8538153,
    "login": "kit101-personal",
    "name": "kit101-personal",
    "url": "https://gitee.com/api/v5/orgs/kit101-personal",
    "avatar_url": "https://portrait.gitee.com/uploads/avatars/namespace/2846/8538153_kit101-personal_1632964073.png?is_link=true",
    "repos_url": "https://gitee.com/api/v5/orgs/kit101-personal/repos",
    "events_url": "https://gitee.com/api/v5/orgs/kit101-personal/events",
    "members_url": "https://gitee.com/api/v5/orgs/kit101-personal/members{/member}",
    "description": "",
    "follow_count": 1
  },
  "user": {
    "id": 1535738,
    "login": "kit101",
    "name": "kit101",
    "avatar_url": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png",
    "url": "https://gitee.com/api/v5/users/kit101",
    "html_url": "https://gitee.com/kit101",
    "remark": ""
  }
}