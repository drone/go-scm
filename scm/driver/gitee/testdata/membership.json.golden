{
  "Active": true,
  "Role": 2
}